ushahidi/platform
# Replace custom dev box with laravel/homestead

This pull request makes the following changes:
- Updating to homestead
- Fixes issues with mysql only_full_group_by sql mode
- Remove search by current_stage (it was broken anyway)

Test these changes by:
- Run `vagrant destroy`
- Switch branches (make sure you destroyed first!)
- Run `vagrant up`
- Run behat tests against the vagrant box

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform/1724)
